packages/connect/src/data/firmwareInfo.ts

```diff
@@ -88,7 +88,9 @@ const isNewer = (release: FirmwareRelease, features: StrictFeatures) => {
 const isRequired = (changelog: ReturnType<typeof getChangelog>) => {
     if (!changelog || !changelog.length) return null;
 
-    return changelog.some(item => item.required);
+    return changelog.some(item => {
+        return item.required;
+    });
```

Comment:
maybe a useless change

Reply:
Absolutelly, this was just for some logging I was doing there. My plan is to clean that, one I consider the PR ready.